### ISSUE TITLE
Firmware repository gone

### ISSUE THREAD

**Issue by The3File** [anjomro/kobra-unleashed#13]
The link for the firmware repository no longer exists, possibly deleted?
Without it, rooting is not possible.
This is not your fault, but i thought i would mention it, as i have not been able to find it anywhere.

**Comment by anjomro** [comment]
You're right, seems like the link isn't working anymore. However a version is saved at the internet archive:

https://web.archive.org/web/20240412214849/https://github.com/ultimateshadsform/Anycubic-Kobra-2-Series-Firmware
